components/Footer.tsx

```diff
@@ -20,8 +38,21 @@ const Footer = () => {
             </address>
           </div>
         </div>
-        <div className="flex justify-center items-center mt-9 lg:mt-0">
-          <Icons />
+        <div >
+          <ol className="flex justify-center items-center mt-9 lg:mt-0">
+            {icons.map((icon,i) => (
+              <li key={i} className="inline-block px-3">
+                <IconLink
```

Comment:
I didn't realize that anchor tags had `aria-label`. Apparently this is a supported option for cases like this: https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8

Let's go ahead and remove the `IconLink` component and instead just wrap the icon in a link with this `aria-label` prop. Please also add the `aria-hidden` prop to the svg to hide it from screen readers. 

```
<Link
  aria-label={icon.altText}
  href={icon.href}
  target="_blank"
  rel="noopener noreferrer"
    >
      <icon.SVGcomponent 
         className="fill-neutral-black hover:fill-main-blue"
         aria-hidden 
       />
</Link>
```